src/Migration/Sources/Csv.php

```diff
@@ -0,0 +1,310 @@
+<?php
+
+namespace Utopia\Migration\Sources;
+
+use Utopia\Database\Database as UtopiaDatabase;
+use Utopia\Migration\Exception;
+use Utopia\Migration\Resource;
+use Utopia\Migration\Resources\Database\Attribute;
+use Utopia\Migration\Resources\Database\Collection;
+use Utopia\Migration\Resources\Database\Database;
+use Utopia\Migration\Resources\Database\Document;
+use Utopia\Migration\Resources\Storage\File;
+use Utopia\Migration\Source;
+use Utopia\Migration\Sources\Appwrite\Reader;
+use Utopia\Migration\Sources\Appwrite\Reader\Database as DatabaseReader;
+use Utopia\Migration\Transfer;
+use Utopia\Storage\Device;
+
+class Csv extends Source
+{
+    private string $filePath;
+
+    /**
+     * format: `{databaseId:collectionId}`
+     */
+    private string $resourceId;
+
+    private Device $deviceForImports;
+
+    private Reader $database;
+
+    public function __construct(
+        string $resourceId,
+        string $filePath,
+        Device $deviceForImports,
```

Comment:
In this context, this should just be a device, if it was in Appwrite alongside other devices, then this name would make sense. Always prefer a single word for names and don't repeat the namespace in the name when existing context is good enough.